project/Dependencies.scala

```diff
@@ -9,7 +9,7 @@ object Dependencies {
   val slf4jVersion = "1.7.5"
   val awsVersion = "1.11.7"
   val faciaVersion = "2.0.2"
-  val capiModelsVersion = "9.24"
+  val capiModelsVersion = "10.1"
```

Comment:
It's worth pinging @kelvin-chappell (just did) as he recently added the `capiModelsVersion` I believe